packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -249,6 +273,14 @@ export default class Meeting extends StatelessSparkPlugin {
     this.screenPeerConnection = screenPeerConnection;
   }
 
+  unsetPeerConnections() {
+    PeerConnectionManager.close(this.mediaPeerConnection);
```

Comment:
we should send an event on peerconnections being shut down

Reply:
Cool. Will add